Shared/messages/MessageFeedsController.h

```diff
@@ -96,6 +98,7 @@ class MessageFeedsController : public Esri::ArcGISRuntime::Toolkit::AbstractTool
   QList<DataListener*> m_dataListeners;
   QString m_resourcePath;
   LocationBroadcast* m_locationBroadcast = nullptr;
+  QVariantList m_messageFeedProperties;
```

Comment:
Can't this be a `QStringList`?

Reply:
It can but that would require more refactoring. The existing code processes it like this. Do you want me to make that change?